src/utils/csrf-token.ts

```diff
@@ -0,0 +1,7 @@
+export function getCSRFToken(): string | null {
```

Comment:
should we move this write/read operation to zm-x-web? like how userAgent is managed

Reply:
it is better if we keep token related logic here, cz it will only be used in api client